__tests__/app.test.js

```diff
@@ -40,4 +40,37 @@ describe("CORE:",() => {
             expect(typeof endpoints).toBe("object")
         })
     })
+
+    describe("Articles:", () => {
+        describe("GET:", () => {
+            it("200: returns requested article", () => {
+                return request(app)
+                .get("/api/articles/1")
+                .expect(200)
+                .then((response) => {
+                    const {
+                        body: { articles },
+                    } = response;
+                    articles.forEach((article) => {
+                        expect(article).toHaveProperty("author");
+                        expect(article).toHaveProperty("title");
+                        expect(article).toHaveProperty("article_id");
+                        expect(article).toHaveProperty("body");
+                        expect(article).toHaveProperty("created_at");
+                        expect(article).toHaveProperty("topic");
+                        expect(article).toHaveProperty("votes");
+                        expect(article).toHaveProperty("article_img_url");
+                    })
+                })
+            })
```

Comment:
We need to make sure we're doing a length check before we iterate through something as it can give us false passing tests.